### PR TITLE
fix(ios): keyboard height on app load

### LINKED ISSUE #6536: bug(ios): Keyman App, keyboard height wrong at startup
**Describe the bug**
In Keyman app, when the keyboard is loaded for the first time with the app launch, height is small. But it changes to correct height after, reselecting the same keyboard, or changing to a different keyboard or with closing the startup "get started" dialogue box.

**To Reproduce**
Launch the Keyman and check the keyboard height, change the keyboard or reselect same keyboard, then check the height

**Expected behavior**

**Screenshots**
1. Video: reselecting same keyboard 
https://user-images.githubusercontent.com/14062593/163605832-dd83ec17-6df4-4fad-9d81-ddb9450c43c1.MOV

2. With startup dialogue box
https://user-images.githubusercontent.com/14062593/163605922-23d86c55-528e-4c7d-9d86-e5e5a2c6c892.MOV

**Keyman for iPhone/iPad/Android:**
 - Device: iPhone 13 pro max
 - OS: iOS 15.4.1
 - Keyman version: 15.0.234(1.0)
 - Target application: Keyman

**Keyboard**
 - Keyboard name: EuroLatin (SIL)
 - Keyboard version: 1.9.6
 - Language name: English

**Additional context**
This bug is independent of keyboard

### PR DESCRIPTION
**Pull request by jahorton**
Fixes #6536.

So... it turns out that simply deferring the first display of the keyboard in-app (triggered by `.becomeFirstResponder`) until after the currently dispatched message on app's internal main loop is complete, the keyboard loads correctly!

While this doesn't clarify _which_ part of keyboard and app UI initialization are the source of the underlying conflict, it's pretty apparent.  The now-deferred `.becomeFirstResponder` call was triggered by a notification from the initializing `KeymanWebViewController` that the host page has loaded, before the keyboard was ever displayed.  Still isn't displayed when it's triggered, though.

### Test environments

- GROUP_IPHONE_SE:  Test on the (notchless) iPhone SE (2nd gen) through Simulator
- GROUP_IPHONE_12:  Test on the (notched) iPhone 12 Pro Max through Simulator
- GROUP_IPAD_PRO: Test on any IPad Pro target available through Simulator.  Any **one** will suffice.
    - Basically, make sure things look fine on tablets, too.

### Test specs

- TEST_COMPARE_SCREENSHOTS

    1. Load the Keyman app.
    2. If the "Get Started" menu displays, activate the "Don't show again" toggle, then force-quit and restart the app.
    3. Once the keyboard loads, take a screenshot.
    4. Rotate the device two times, in opposite directions.
    5. Take another screenshot.
    6. This test passes **_if and only if_** the keyboard height matches in both screenshots.

While not strictly necessary, it would be nice to have a screenshot of the two screenshots lined up, side by side, to facilitate an "eyeball test" when reviewing the results.

An example, **failing** version of this:  from 'before' this PR's changes...

image

(Left = before, right = after)

### CODE REVIEW

File: ios/keyman/Keyman/Keyman/Classes/MainViewController/MainViewController.swift [hunk 1]
```diff
@@ -464,11 +470,10 @@ class MainViewController: UIViewController, TextViewDelegate, UIActionSheetDeleg
       navigationItem.titleView = nil
       navigationItem.rightBarButtonItem = nil
       setNavBarButtons()
-      if wasKeyboardVisible {
-        perform(#selector(self.displayKeyboard), with: nil, afterDelay: 0.75)
-      }
       if shouldShowGetStarted {
         perform(#selector(self.showGetStartedView), with: nil, afterDelay: 0.75)
+      } else if wasKeyboardVisible {
+        perform(#selector(self.displayKeyboard), with: nil, afterDelay: 0.75)
```
Comment (jahorton) [inline ios/keyman/Keyman/Keyman/Classes/MainViewController/MainViewController.swift:476]: Note:  showing 'get started' will hide the keyboard immediately anyway.  This change just drops a little unnecessary busy-ness.

**Review by sgschantz — APPROVED** [review]
lgtm

**Comment by jahorton** [comment]
Re: testing - that may be blocked on #6673 at the moment.

**Comment by bharanidharanj** [comment]
### GROUP_IPHONE_SE: Test on the (notchless) iPhone SE (2nd gen) through Simulator

* **TEST_COMPARE_SCREENSHOTS (PASSED):** Tested this as per the instructions and it seems that the height remains the same  after rotation. 


### GROUP_IPHONE_12: Test on the (notched) iPhone 12 Pro Max through Simulator

* **TEST_COMPARE_SCREENSHOTS (PASSED):** Tested this as per the instructions and it seems that the height remains the same  after rotation. 



### GROUP_IPAD_PRO: Test on any IPad Pro target available through Simulator.  Any **one** will suffice.

* **TEST_COMPARE_SCREENSHOTS (PASSED):** Tested this as per the instructions in IPad Pro (15.0 OS) Simulator and it seems that the height remains the same  after rotation.

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.254-beta](https://keyman.com/downloads/releases/beta/15.0.254)
